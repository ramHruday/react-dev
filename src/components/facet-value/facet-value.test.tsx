import { shallow } from 'enzyme';
import React from 'react';

import FacetValue from './facet-value';

describe('FacetValue', () => {
  it('should render FacetValue', () => {
    const component = shallow(<FacetValue facet={{ count: 10, value: 'something' }} isSelected />);
    expect(component).toMatchSnapshot();
  });
});
