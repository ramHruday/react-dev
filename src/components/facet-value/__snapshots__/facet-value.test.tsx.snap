// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FacetValue should render FacetValue 1`] = `ShallowWrapper {}`;
